ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -919,13 +1016,34 @@ bb_decode (void **pdata, const GstTensorsConfig * config,
       default:
         g_assert (0);
     }
+  } else if (bdata->mode == OV_PERSON_DETECTION_BOUNDING_BOX) {
+    results = g_array_sized_new (FALSE, TRUE, sizeof (detectedObject),
+        OV_PERSON_DETECTION_MAX);
+
+    /* Already checked with getOutCaps. Thus, this is an internal bug */
+    g_assert (num_tensors >= OV_PERSON_DETECTION_MAX_TENSORS);
+
+    switch (config->info.info[0].type) {
+      _get_persons_ov(bdata, uint8_t, input[0].data, _NNS_UINT8, results);
+      _get_persons_ov(bdata, int8_t, input[0].data, _NNS_INT8, results);
+      _get_persons_ov(bdata, uint16_t, input[0].data, _NNS_UINT16, results);
+      _get_persons_ov(bdata, int16_t, input[0].data, _NNS_INT16, results);
+      _get_persons_ov(bdata, uint32_t, input[0].data, _NNS_UINT32, results);
+      _get_persons_ov(bdata, int32_t, input[0].data, _NNS_INT32, results);
+      _get_persons_ov(bdata, uint64_t, input[0].data, _NNS_UINT64, results);
+      _get_persons_ov(bdata, int64_t, input[0].data, _NNS_INT64, results);
+      _get_persons_ov(bdata, float, input[0].data, _NNS_FLOAT32, results);
+      _get_persons_ov(bdata, double, input[0].data, _NNS_FLOAT64, results);
+      default:
+        g_assert (0);
+    }
   } else {
     GST_ERROR ("Failed to get output buffer, unknown mode %d.", bdata->mode);
     return GST_FLOW_ERROR;
   }
 
   draw (&out_info, bdata, results);
-  g_array_free (results, TRUE);
+  g_array_free (results, FALSE);
```

Comment:
Why?

Reply:
@myungjoo It seems that ```results``` holds its own memory spaces for each element and ```g_array_append_val``` just copies the data to the internal memory spaces. I think that it is enough to just call ```g_array_free``` without setting the second parameter unless we append a container of something.

Reply:
Ah.. got it..